tests/authcrypt.log

```diff
@@ -0,0 +1,4 @@
+plaintext message: [0-9a-fA-F]*
+ciphertext: [0-9a-fA-F]*
+decrypted: [0-9a-fA-F]*

```

Comment:
Sorry, my previous message was a bit misleading because I thought this was hashing instead of authcrypt. I still think bits of it do not need a complex regexp. For example the plaintext message and the decrypted message must be the same value and is hardcoded in the example.